Tools/CISupport/ews-build/steps.py

```diff
@@ -4367,70 +4363,65 @@ class RunAPITests(TestWithFailureCount):
     def __init__(self, **kwargs):
         super(RunAPITests, self).__init__(logEnviron=False, **kwargs)
 
-    def start(self):
+    @defer.inlineCallbacks
+    def run(self):
         platform = self.getProperty('platform')
         if platform == 'gtk':
-            command = ['python3', 'Tools/Scripts/run-gtk-tests',
+            self.command = ['python3', 'Tools/Scripts/run-gtk-tests',
                        '--{0}'.format(self.getProperty('configuration')),
                        '--json-output={0}'.format(self.jsonFileName)]
```

Comment:
Those indentation also need to be updated?
Also if this is python3 only, we can do 
```
...
'--json-output={0}'.format(self.jsonFileName), *customBuildFlag(platform, self.getProperty('fullPlatform'))]
```